src/redux/categories/categories.js

```diff
@@ -0,0 +1,22 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As it is in the project requirements, kindly note that the action types are supposed t be implemented using the [duck's pattern](https://github.com/erikras/ducks-modular-redux). Kindly implement this.